test/cmocka/src/audio/pipeline/pipeline_connect_upstream.c

```diff
@@ -108,11 +108,8 @@ static void test_audio_pipeline_complete_connect_downstream_variable_set
 	/*Testing component*/
 	pipeline_complete(&result, test_data->first, test_data->second);
 
-	assert_int_equal
-	(
-	result.sched_comp->frames, test_data->p.ipc_pipe.frames_per_sched
-	);
 	assert_ptr_equal(result.sched_comp->pipeline, &result);
+	assert_ptr_equal(test_data->first->pipeline, &result);
```

Comment:
Typo in commit description: `successful`.

Reply:
ok